test/e2e/provisioning/testdata/machinedeployment-nutanix.yaml

```diff
@@ -33,6 +33,8 @@ spec:
             clusterName: '<< NUTANIX_CLUSTER >>'
             projectName: '<< NUTANIX_PROJECT >>'
             subnetName: '<< NUTANIX_SUBNET >>'
+            additionalSubnetNames:
+              - '<< NUTANIX_ADDITIONALSUBNETS >>'
```

Comment:
Our e2e environment currently does not have multiple subnets, so I'm afraid we won't be able to test for this anyway. I would say we remove the references from `test/e2e/provisioning` for now and will revisit this on our end once we have the ability to integrate it into the test environment. You can however provider `additionalSubnetNames: []` in this file if you want, to. make sure we test with the field present.

Reply:
changed to additionalSubnetNames: [] as suggested